<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestCdnSetting</class>
 <widget class="QDialog" name="TestCdnSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDN混流设置</string>
  </property>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>直播流ID：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoWidth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>211</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关键帧间隔：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mode">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>非法值</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>全手动模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>纯音频模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>预排版模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>屏幕分享模式</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>91</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音频码率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>251</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率的高度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bizId">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>291</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3891</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>131</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音频声道数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>211</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率的宽度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="audioBitrate">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>91</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>64</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>91</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频码率：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="streamId">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>321123</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>291</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>直播 bizid：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="audioChannels">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>131</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>混流参数配置模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>131</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频帧率：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startPublishingBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋_GB2312</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string> 开始/取消推流</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoFramerate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>131</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音频采样率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STFangsong</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请在下方依次设置云端混流转码的参数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backgroundImage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>51</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>您的图片ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoGOP">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoHeight">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>251</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>混合后画面背景图：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="audioSampleRate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>12000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>44100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>48000</string>
    </property>
   </item>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>2</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSlider" name="videoBitrate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>80</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>550</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="videoBitrateLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>550 kbps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateMixTranscodingConfigBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置混流转码参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋_GB2312</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数变化请点击设置按钮一次性更新：</string>
   </property>
  </widget>
  <zorder>listView_2</zorder>
  <zorder>listView</zorder>
  <zorder>label_16</zorder>
  <zorder>videoWidth</zorder>
  <zorder>label_10</zorder>
  <zorder>mode</zorder>
  <zorder>label_15</zorder>
  <zorder>label_7</zorder>
  <zorder>bizId</zorder>
  <zorder>label_14</zorder>
  <zorder>label_6</zorder>
  <zorder>audioBitrate</zorder>
  <zorder>label_9</zorder>
  <zorder>streamId</zorder>
  <zorder>label_5</zorder>
  <zorder>audioChannels</zorder>
  <zorder>label_4</zorder>
  <zorder>label_8</zorder>
  <zorder>startPublishingBtn</zorder>
  <zorder>videoFramerate</zorder>
  <zorder>label_13</zorder>
  <zorder>label_17</zorder>
  <zorder>backgroundImage</zorder>
  <zorder>videoGOP</zorder>
  <zorder>videoHeight</zorder>
  <zorder>label_11</zorder>
  <zorder>audioSampleRate</zorder>
  <zorder>videoBitrate</zorder>
  <zorder>videoBitrateLabel</zorder>
  <zorder>updateMixTranscodingConfigBtn</zorder>
  <zorder>label_18</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
